pkg/apiserver/apiserver.go

```diff
@@ -74,13 +72,13 @@ type APIServer struct {
 	aggClient aggregatorclient.Interface
 }
 
-func NewAPIServer(clientConfig *rest.Config, coreClient kubernetes.Interface, kcClient kcclient.Interface, globalNamespace string, bindPort int) (*APIServer, error) {
+func NewAPIServer(clientConfig *rest.Config, coreClient kubernetes.Interface, kcClient kcclient.Interface, globalNamespace string, bindPort int, enableAPIPriorityAndFairness bool) (*APIServer, error) {
```

Comment:
Something we'll want to consider in the long term is a better place for the controller options (e.g. globalNamespace and enableAPIPriorityAndFairness). Ideally, we would just pass in the whole opts struct versus continuing to add more params to this func. I think we don't do that now cause of the circular import that comes up from importing controller.Options here.

Reply:
yeah there's gotta be some german word for like, "feeling guilty because you added one more parameter to an already long function signature". right?   

maybe "schadenparameterverbreiten" which is roughly "shameful parameter spreading" ? 

anyways that's how i felt adding this param.  Agree it's getting a bit ugly.